nexus/src/db/queries/network_interface.rs

```diff
@@ -30,6 +30,51 @@ use omicron_common::api::external;
 use std::net::IpAddr;
 use uuid::Uuid;
 
+// These are sentinel values and other constants used to verify the state of the
+// system when operating on network interfaces
+lazy_static::lazy_static! {
```

Comment:
Good point, thanks. I believe that's a holdover from when these were consts inside the struct(s) that needed them, rather than shared between two or more queries.